tools/macosx-firewall.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Script that adds rules to Mac OS X Socket Firewall to avoid
+# popups asking to accept incoming network connections when
+# running tests.
+SFW="/usr/libexec/ApplicationFirewall/socketfilterfw"
+TOOLSDIR="`dirname \"$0\"`"
+TOOLSDIR="`( cd \"$TOOLSDIR\" && pwd) `"
+ROOTDIR="`( cd \"$TOOLSDIR/..\" && pwd) `"
+OUTDIR=$TOOLSDIR/../out
+OUTDIR="`( cd \"$OUTDIR\" && pwd) `"
```

Comment:
It is really so that the path added is the full path without the `..` in it. This is what is displayed when the command is run (if using `..` in that path that is):
```shell
Application at path ( /Users/danielbevenius/work/nodejs/node/tools/../out/Debug/node ) added to firewall
```
But the entry when running `/usr/libexec/ApplicationFirewall/socketfilterfw --listapps` shows up without the path. Running the script again will not remove these paths, instead the new rules will just be added and the list will grow. But with the full path (without `..`) it works as expected and the rules are removed and added properly.